src/test/java/com/ericsson/ei/jmespath/test/TestStringSplittingFunction.java

```diff
@@ -37,7 +37,10 @@ public void testStringSplit() throws Exception {
 
     @Test
     public void testStringSplitFunction() throws Exception {
-        String processRule = "split(data.identity, '/')";
+        // String processRule = "split(data.identity, '/')";
+
+        String processRule = "split(data.identity, '@|/')";
```

Comment:
Why this change?

Reply:
it was unintended merge.... fixed in next commit

Reply:
I removed it as it was not required.